.expeditor/config.yml

```diff
@@ -86,6 +86,8 @@ subscriptions:
       - bash:.expeditor/update_chef-analyze_to_latest.sh
   - workload: artifact_published:stable:chef:16*
     actions:
+      # hack to make sure update_dep creates a proper PR title
+      - export EXPEDITOR_GEM_NAME="Chef Infra Client"
```

Comment:
So, this won't work. 

Could we maybe switch to subscribing to the gem publish? It would functionally be the same.

Reply:
I think the reason we didn't do the gem publish is that we want to subscribe to just chef (not chef-*) and not chef 15. In the current config we pin to the chef 16 promotes.